### PR TITLE
Feature analyze by sentence

### PR DESCRIPTION
**Pull request by lahnerml**
To find 'bad' sentences I decided to implement a per_sentence_score method.

### CODE REVIEW

File: lib/odyssey.rb [hunk 1]
```diff
@@ -60,5 +60,5 @@ def self.to_ary
 
 require 'require_all'
 require 'odyssey/engine'
-require_rel 'formulas/formula'
+# require_rel 'formulas/formula'
```
Comment (cameronsutter) [inline lib/odyssey.rb]: You'll have to remove this file because of your previous PR

**Comment by cameronsutter** [comment]
Besides the merge conflict, it looks good

**Comment by lahnerml** [comment]
Done.

**Comment by cameronsutter** [comment]
It's still saying there's conflicts. I made a new PR: https://github.com/cameronsutter/odyssey/pull/14

**Comment by lahnerml** [comment]
Merged #14 and closing this one.

**Pull request closed without merging**